<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Backup system simulation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCalculate">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate!</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckBxControllerActive">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>583</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Controller Active</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckBxFireDetected">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>136</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire detected</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time1Fire">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblStartSec">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>72</width>
      <height>39</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>start (sec):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEndSec">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>65</width>
      <height>39</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>end (sec):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time2Fire">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTotalPostlubeTime">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>137</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Total postlube time:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalPostlubeHours">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHoursAnd">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>hour(s) and </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalPostlubeMinutes">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMinutes">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>minutes</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckBxHeaderLo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Header pressure low</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time1Header">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="time2Header">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblEndSec_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>65</width>
      <height>39</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>end (sec):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStartSec_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>72</width>
      <height>39</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>start (sec):</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
